src/slider/slider.scss

```diff
@@ -0,0 +1,222 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+@import '../icon/index';
+
+$thumb-height: $iui-m;  // 16px
+$track-height: $iui-xs;  // 4px
+$tick-height: ($iui-s + $iui-xxs);
```

Comment:
This is 10px... Any reason? It is just not really up to standards.. Could we change it to 8 or 12..?

Reply:
It needs to be greater than 8, since thumb has radius of 8 which hides the tick.   12 is OK, just bigger than it needs to be.  Shouldn't this be a decision for Jon?

Reply:
I changed `$tick-height` to use `$iui-baseline` = 11px.  Baseline is intended for spacing on the Y axis, so I think it makes sense here.